clippy_lints/src/tuple_array_conversions.rs

```diff
@@ -31,7 +31,7 @@ declare_clippy_lint! {
     /// ```
     #[clippy::version = "1.72.0"]
     pub TUPLE_ARRAY_CONVERSIONS,
-    complexity,
+    pedantic,
```

Comment:
This should be `nursery` instead.

I'll try to fix most of these issues before the next sync (in 2 weeks).

Reply:
Ok